build.sbt

```diff
@@ -108,6 +108,7 @@ lazy val commonSettings = testSettings ++
     s3("downloads.mesosphere.io/maven")
   )),
   s3credentials := new EnvironmentVariableCredentialsProvider() | InstanceProfileCredentialsProvider.getInstance(),
+  s3region := com.amazonaws.services.s3.model.Region.US_West_2,
```

Comment:
Is it the right region?

Reply:
It used to be a different one:

https://github.com/ohnosequences/sbt-s3-resolver/blob/v0.8.0/src/main/scala/SBTS3Resolver.scala#L51